Resources/Prototypes/Datasets/ion_storm.yml

```diff
@@ -363,12 +365,12 @@
   - GARGLE BLASTERS
   - LEAN
   - LONG ISLAND ICED TEA
-  - NUKA COLA
+  - PIÑA COLADA
```

Comment:
Obviously nuka cola isn't in the game, and this means a borg has a reason to sing the Pina Colada song or something :>